apps/video-demo/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - GoogleAds-IMA-iOS-SDK (3.14.4)
-  - Mux-Stats-AVPlayer (2.9.0):
+  - Mux-Stats-AVPlayer (2.10.0):
     - Mux-Stats-Core (~> 3.8)
   - Mux-Stats-Core (3.8.0)
   - Mux-Stats-Google-IMA (0.6.0):
```

Comment:
shouldn't this be 0.7.0?

Reply:
How are the PiP views generated in CI looking? I'm assuming it's not working? Maybe we should add a step to check those to the README since it's totally a manual sanity check right now :)

Reply:
I assume this means the PiP views in the CI environment aren't working? We should probably add a step to check those before releasing in the README, since it is just a manual sanity check right :)

Reply:
the ima sdk 0.7.0 release has a dependency on this sdk so in order to make that release we need to first get this release done. That is why we had to extract the examples logic that would have required ima sdk 0.7.0. On the next release we can add the example and update the podfile locks to target the new ima version

Reply:
ohh right I remember this discussion now - thanks